### PR TITLE
Clarify Cosmovisor docs; suggest using UNSAFE_SKIP_BACKUP

### PR DESCRIPTION
**Pull request by BrendanChou**

### CODE REVIEW

File: pages/validators/upgrades/performing_upgrades.md [hunk 1]
```diff
@@ -1,29 +1,18 @@
-# Performing upgrades
+# Performing Upgrades
 
-## Running a validator and managing upgrades
+## Managing Upgrades
```
Comment (BrendanChou) [inline pages/validators/upgrades/performing_upgrades.md:3]: removed "validator" wording as this is not validator-specific

File: pages/validators/upgrades/performing_upgrades.md [hunk 2]
```diff
@@ -1,29 +1,18 @@
...
-`cosmovisor` is a small process manager for Cosmos SDK application binaries that monitors the governance module for incoming chain upgrade proposals. If it sees a proposal that gets approved, `cosmovisor` can automatically download the new binary, stop the current binary, switch from the old binary to the new one, and finally restart the node with the new binary.
-
-We recommend validators to use `cosmovisor` to run their nodes. This will make low-downtime upgrades smoother, as validators don’t have to manually upgrade binaries during the upgrade. Instead, they can pre-install new binaries, and `cosmovisor` will automatically update them based on the on-chain software upgrade proposals.
-
-### Installation instructions
-
-See [here](../cosmovisor.md)  for installation instructions.
-
----
-
```
Comment (BrendanChou) [inline pages/validators/upgrades/performing_upgrades.md:19]: moved to Cosmovisor page

File: pages/validators/_meta.json [hunk 3]
```diff
@@ -3,7 +3,6 @@
   "required_node_configs": "Required Node Configs",
   "running_validator": "Running a Validator",
   "running_full_node": "Running a Full Node",
-  "cosmovisor": "Setting up Cosmovisor",
```
Comment (BrendanChou) [inline pages/validators/_meta.json:6]: moved to the upgrades section/folder

File: pages/validators/upgrades/cosmovisor.md [hunk 4]
```diff
@@ -1,4 +1,17 @@
-# Setting up Cosmovisor
+# Cosmovisor
+
+`cosmovisor` is a small process manager for Cosmos SDK application binaries that monitors the governance module for incoming chain upgrade proposals. If it sees a proposal that gets approved, `cosmovisor` can automatically download the new binary, stop the current binary, switch from the old binary to the new one, and finally restart the node with the new binary.
+
+We recommend validators to use `cosmovisor` to run their nodes. This will make low-downtime upgrades smoother, as validators don’t have to manually upgrade binaries during the upgrade. Instead, they can pre-install new binaries, and `cosmovisor` will automatically update them based on the on-chain software upgrade proposals.
+
+## Configuration
+
+When Cosmovisor activates an upgrade, it does a backup of the entire data directory by default. This backup can take a very long time to process unless the user does agressive historical-state-pruning using the `pruning` [configuration on the node](../required_node_configs.md).
```
Comment (roy-dydx) [inline pages/validators/upgrades/cosmovisor.md:9]: s/agressive/aggressive/

**Review by roy-dydx — APPROVED** [review]

**Pull request merged**